src/gui/screen_printing.cpp

```diff
@@ -779,11 +743,49 @@ void set_stop_icon_and_label(screen_t *screen) {
 }
 
 void change_print_state(screen_t *screen, printing_state_t st) {
-    _dbg("printstate %d entered", (int)st);
+    //_dbg("printstate %d entered", (int)st);
```

Comment:
is this comment block still relevant?

Reply:
I commented this line out because it filled dbg output too much. This source will be refactored soon, we can keep commented/dead code for now.